bedrock/mozorg/templates/mozorg/about/leadership.html

```diff
@@ -93,32 +93,26 @@ <h3 class="group-title" id="mgmt-corp"><a href="{{ url('foundation.moco') }}">{{
+            Mitchell is a member of the Advisory Boards of the Oxford Internet Institute and the MIT Initiative on the Digital Economy. She is an MIT Media Lab Research Affiliate with the Open Agriculture Initiative, and a Board Member of OpenMRS, Inc., which develops open source medical records systems for use in resource-constrained environments. She co-chaired the U.S. Department of Commerce Digital Economy Board of Advisors from its inception in March 2016 until August 2017, served on the United Nations High Level Panel on Women’s Economic Empowerment, and the ICANN High Level Panel on Global Internet Cooperation and Governance Mechanisms. A native of the San Francisco Bay Area, Mitchell received her B.A. in Asian Studies from UC Berkeley and her J.D. from Berkeley Law.
           {% endtrans %}
           </p>
 
           <p>
             <em>
             {% trans %}
-              TIME Magazine profiled Mitchell in its global list of “100 Most Influential People” under “Scientists and Thinkers”. Bloomberg listed her as one of the “25 Most Influential People on the Web”. She was honored as winner of the Anita Borg Institute’s 2009 Women of Vision Award and received the Aenne Burda Award for Creative Leadership in 2010. In 2012, Mitchell was inducted into the founding group of the <a rel="external" href="http://www.internethalloffame.org/inductees/mitchell-baker">Internet Society’s Hall of Fame</a>. In 2018 she received an honorary doctorate from the Universite Catholique de Louvain in Belgium. She has appeared on NBC “Meet the Press”, BBC “HardTalk”, “CNN Global Office” and "NPR Morning Edition". She has spoken at high-level events like the World Economic Forum, Mobile World Congress, Web Summit, Collision, Singularity University and Bits&Pretzels.
+              TIME Magazine profiled Mitchell in its global list of “100 Most Influential People” under “Scientists and Thinkers”. Bloomberg listed her as one of the “25 Most Influential People on the Web”. She was honored as winner of the Anita Borg Institute’s 2009 Women of Vision Award and received the Aenne Burda Award for Creative Leadership in 2010. In 2012, Mitchell was inducted into the founding group of the <a rel="external" href="http://www.internethalloffame.org/inductees/mitchell-baker">Internet Society’s Hall of Fame</a>. In 2018 she received an honorary doctorate from the Universite Catholique de Louvain in Belgium. She has written op-eds for the Atlantic and CNN.com and appeared on NBC “Meet the Press”, BBC “HardTalk”, “CNN Global Office”, NBC’s “Weekend Today”, France 24, the UK's Channel 4 and NPR’s “Morning Edition”. She has spoken at high-level events like the World Economic Forum, Mobile World Congress, Web Summit, Collision and Singularity University.
```

Comment:
Just noting minor inconsistencies in this paragraph, e.g. `NBC "Meet the Press"` separated and different from `NBC’s “Weekend Today”`, and `BBC “HardTalk”` contrasted with `“CNN Global Office”`.

Reply:
Looks like it's just "Global Office" so that one can be possessive as well. Will update.